src/components/dashboard/Dashboard.js

```diff
@@ -134,112 +133,169 @@ const Dashboard = props => {
-          )}
-          <Section.Row style={styles.buttonsRow} alignItems="stretch">
-            <PushButton
-              routeName="Who"
-              screenProps={screenProps}
-              style={styles.leftButton}
-              icon="send"
-              iconAlignment="left"
-              params={{
-                nextRoutes: ['Amount', 'Reason', 'SendLinkSummary', 'SendConfirmation'],
-                params: { action: 'Send' },
-              }}
-            >
-              Send
-            </PushButton>
-            <ClaimButton screenProps={screenProps} amount={weiToMask(entitlement, { showUnits: true })} />
-            <PushButton
-              routeName={'Receive'}
-              screenProps={screenProps}
-              style={styles.rightButton}
-              icon="receive"
-              iconAlignment="right"
-            >
-              Receive
-            </PushButton>
-          </Section.Row>
-        </Section>
-        <FeedList
-          handleFeedSelection={handleFeedSelection}
+    <Wrapper style={styles.dashboardWrapper}>
+      <Section style={[styles.topInfo]}>
+        {scrollPos < 100 ? (
+          <Section style={[styles.userInfo, styles.userInfoVertical]}>
+            <Avatar onPress={() => screenProps.push('Profile')} size={68} source={avatar} style={[styles.avatarBig]} />
+            <Section.Title style={[styles.userName]}>{fullName || ' '}</Section.Title>
+            <BigGoodDollar
+              bigNumberStyles={styles.bigNumberVerticalStyles}
+              bigNumberUnitStyles={styles.bigNumberUnitStyles}
+              number={balance}
+            />
+          </Section>
+        ) : (
+          <Section style={[styles.userInfo, styles.userInfoHorizontal]}>
+            <Avatar
+              onPress={() => screenProps.push('Profile')}
+              size={42}
+              source={avatar}
+              style={[styles.avatarSmall]}
+            />
+            <BigGoodDollar
+              bigNumberStyles={styles.bigNumberStyles}
+              bigNumberUnitStyles={styles.bigNumberUnitStyles}
+              number={balance}
+            />
+          </Section>
+        )}
+        <Section.Row style={styles.buttonsRow}>
+          <PushButton
+            icon="send"
+            iconAlignment="left"
+            routeName="Who"
+            screenProps={screenProps}
+            style={[styles.leftButton, { elevation: 0 }]}
+            params={{
+              nextRoutes: ['Amount', 'Reason', 'SendLinkSummary', 'SendConfirmation'],
+              params: { action: 'Send' },
+            }}
+          >
+            Send
+          </PushButton>
+          <ClaimButton screenProps={screenProps} amount={weiToMask(entitlement, { showUnits: true })} />
+          <PushButton
+            icon="receive"
+            iconAlignment="right"
+            routeName={'Receive'}
+            screenProps={screenProps}
+            style={[styles.rightButton, { elevation: 0 }]}
```

Comment:
Can we add elevation in rightButton class?

Reply:
I'll check again if it works, it might have been hot reload not working properly.

Reply:
It worked. Moving it into the class.